pallets/runtime/src/bridge.rs

```diff
@@ -357,6 +367,28 @@ decl_module! {
             ensure!(account_key == Self::admin_key(), Error::<T>::Unauthorized);
             Ok(())
         }
+
+        /// Emits an event containing the timelocked balances of a given `IssueRecipient`.
+        pub fn get_timelocked_balances_of_recipient(
```

Comment:
I think for this RPC call is more preferable. What we can do is add this function as a module function instead of dispatchable and then in the next task we will convert that into an RPC call.